carlsim/kernel/inc/snn.h

```diff
@@ -861,6 +866,40 @@ class SNN {
 	void spikeGeneratorUpdate_CPU(int netId);
 	void updateTimingTable_CPU(int netId);
 	void updateWeights_CPU(int netId);
+#else // POSIX
```

Comment:
Maybe a comment on what's exactly different between these files and the above? Is it only the return type?